service_capacity_modeling/models/utils.py

```diff
@@ -31,11 +38,20 @@ def reduce_by_family(plans: Iterable[CapacityPlan]) -> List[CapacityPlan]:
                 sorted({(c.cluster_type, c.instance.family) for c in topo.zonal})
             )
 
-        if not (zonal_type in zonal_families and regional_type in regional_families):
+        # Count how many of each family combination we've seen
+        zonal_count = zonal_families.get(zonal_type, 0)
+        regional_count = regional_families.get(regional_type, 0)
+
+        # Add the plan if we haven't reached the maximum for either family type
+        if (
+            zonal_count < max_results_per_family
```

Comment:
If max_results_per_family is 3 then according to this we can have 3 zonal count and 3 regional count. Is it expected or will it  be 3 for zonal and regional combined?